src/syscalls.py

```diff
@@ -1336,7 +1336,7 @@ def __init__(self, **kwargs):
 
 vserver = InvalidSyscall(x86=273, x64=236)
 mbind = EmulatedSyscall(x86=274, x64=237)
-get_mempolicy = EmulatedSyscall(x86=275, x64=239)
+get_mempolicy = EmulatedSyscall(x86=275, x64=239, arg1="signed int", arg2="unsigned long")
```

Comment:
Sorry, another thing: The latter should be `typename Arch::unsigned_long` because that is different for x86 vs x86-64.